portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,81 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Mariafernanda Salas</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1>Mariafernanda Salas</h1>
+      
+      <h2>About Me</h2>
+      <img src="/images/IMG_9818.JPG" alt="MariferSalas" style="float:left;width:220px; height:220px; padding-right:20px;"/>
```

Comment:
Please add the style here into the CSS file and for the other instances as well. Inline CSS makes it harder to maintain sites, especially when collaborating with others. Especially so if you have a mix of inline and a separate CSS document because it makes it difficult to debug where the CSS issues are coming from. I can see several instances below where you use the same style "text-align: center", where you can create a class and apply it across.

Reply:
Thank you for that, it will be easier later to change things directly from the CSS file.